test/explorer/data_frame_test.exs

```diff
@@ -2034,4 +2034,16 @@ defmodule Explorer.DataFrameTest do
                    fn -> DF.put(df, :e, Nx.tensor([1, 2, 3], type: {:u, 32})) end
     end
   end
+
+  test "describe/1" do
+    df = DF.new(a: ["d", "e", "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```

Comment:
Let's add a nil to one of the columns to see how it affects the result? In particular, I am wondering if count will decrease:

```suggestion
    df = DF.new(a: ["d", nil, "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```